src/current/_includes/common/tls-bad-cipher-warning.md

```diff
@@ -0,0 +1,14 @@
+{{site.data.alerts.callout_info}}
+CockroachDB clusters can talk TLS 1.2 or TLS 1.3 to SQL clients for encryption-in-flight and to prove each other's identities. However, the following less-secure TLS 1.2 cipher suites are rejected, in accordance with the IETF's recommended cipher list defined in [RFC 8447](https://datatracker.ietf.org/doc/html/rfc8447):
```

Comment:
Perhaps I'm not up-to-speed with the security lingo, but "talk" sounds a little odd. Would switching out for "can talk" to "support" work?